### DISCUSSION TITLE
MissingConfigException: In 'modulus_default': Could not find 'amp/default'

### DISCUSSION THREAD

**Discussion by moduluslover** [NVIDIA/physicsnemo#747] [General]
```

 import modulus.sym
from modulus.sym.hydra import to_yaml
from modulus.sym.hydra.utils import compose
from modulus.sym.hydra.config import ModulusConfig

cfg = compose(config_path=".", config_name="config")
cfg.network_dir = 'outputs'    # Set the network directory for checkpoints
print(to_yaml(cfg))
```

   490     assert loaded is not None

/usr/local/lib/python3.10/dist-packages/hydra/_internal/defaults_list.py in config_not_found_error(repo, tree)
    797     msg += "\nConfig search path:" + f"\n{lines}"
    798 
--> 799     raise MissingConfigException(
    800         missing_cfg_file=element.get_config_path(),
    801         message=msg,

MissingConfigException: In 'modulus_default': Could not find 'amp/default'

Config search path:
	provider=hydra, path=pkg://hydra.conf
	provider=main, path=file:///kaggle/working
	provider=schema, path=structured://

**Comment by ram-cherukuri** [comment]
@moduluslover Is this still an issue that you need help with?
